Parser/Device/AbstractDeviceParser.php

```diff
@@ -329,6 +329,7 @@ abstract class AbstractDeviceParser extends AbstractParser
         'C3'  => 'China Mobile',
         'U9'  => 'China Telecom',
         'CI'  => 'Chico Mobile',
+        'CIA' => 'Cidea',
```

Comment:
Based on photos I've found (for example https://jiji.ng/surulere/tablets/new-c-idea-cm7000-plus-512-gb-rsYzAEHJMsdZor3u7ZfnlZks.html) it should be `C Idea`